src/main/kotlin/net/atos/zac/app/informatieobjecten/EnkelvoudigInformatieObjectUpdateService.kt

```diff
@@ -62,7 +61,8 @@ class EnkelvoudigInformatieObjectUpdateService @Inject constructor(
     fun createZaakInformatieobjectForZaak(
         zaak: Zaak,
         enkelvoudigInformatieObjectCreateLockRequest: EnkelvoudigInformatieObjectCreateLockRequest,
-        taskId: String? = null
+        taskId: String? = null,
+        skipPolicyCheck: Boolean = false,
```

Comment:
I don't see any calling code in this PR where this parameter is set to `true` or am I missing things? Also it seems from the PR title that this PR is really about something else instead of skipping policy checks? I'm a bit confused. ;-)

Reply:
Have a look at https://github.com/infonl/dimpact-zaakafhandelcomponent/pull/2178/files#diff-5a46a83910f7bf5f25310415d621ced47b129d04ed1bd199c879ee385ebf3a6bR127